src/main/java/edu/uc/javahotkey/JavaHotKeyController.java

```diff
@@ -25,10 +25,10 @@ public Project sampleJsonSchema() {
         project.setName("Sample Project");
         KeyMap keyMap = new KeyMap();
         keyMap.setFunctionName("func_name");
-        keyMap.setKeymap(new ArrayList<Integer>());
+        keyMap.setKeymap(new ArrayList<>());
```

Comment:
Removed the type because you don't need to define it in Java.

Reply:
Well... it doesn't hurt.  Especially here, where you are creating an object inline.